centml/compiler/server.py

```diff
@@ -6,26 +6,23 @@
 from fastapi import FastAPI, UploadFile, HTTPException, BackgroundTasks, Response
 from fastapi.responses import FileResponse
 from fastapi.middleware.gzip import GZipMiddleware
-from centml.compiler.server_compilation import hidet_backend_server, storage_path, dir_cleanup
+from centml.compiler.server_compilation import hidet_backend_server
+from centml.compiler.utils import dir_cleanup
 from centml.compiler.config import config_instance, CompilationStatus
-
-logger = logging.getLogger(__name__)
+from centml.compiler.utils import get_server_compiled_forward_path
 
 app = FastAPI()
 app.add_middleware(GZipMiddleware, minimum_size=1000)
 
 
 def get_status(model_id: str):
-    if not os.path.isdir(os.path.join(storage_path, model_id)):
+    if not os.path.isdir(os.path.join(config_instance.SERVER_BASE_PATH, model_id)):
         return CompilationStatus.NOT_FOUND
 
-    if not os.path.isfile(os.path.join(storage_path, model_id, "cgraph.zip")):
+    if not os.path.isfile(get_server_compiled_forward_path(model_id)):
         return CompilationStatus.COMPILING
 
-    if os.path.isfile(os.path.join(storage_path, model_id, "cgraph.zip")):
-        return CompilationStatus.DONE
-
-    return None
+    return CompilationStatus.DONE
```

Comment:
why did we remove the condition check?

Reply:
The condition before it returns from the function if the condition I removed isn't met. 
This means if we get to the line `return CompilationStatus.DONE` then the condition `os.path.isfile(os.path.join(storage_path, model_id, "cgraph.zip"))` is already met, so the condition check I removed is useless.